java/vector/src/main/java/org/apache/arrow/vector/BaseVariableWidthVector.java

```diff
@@ -792,15 +791,15 @@ private void splitAndTransferValidityBuffer(int startIndex, int length,
   }
 
 
-  /******************************************************************
-   *                                                                *
-   *                common getters and setters                      *
-   *                                                                *
-   ******************************************************************/
+  /*----------------------------------------------------------------*
```

Comment:
Lol, just curious, which rule is not happy about the ascii art? (Feel free to ignore this comment btw

Reply:
Yeah, it thinks its a Javadoc and gives all kinds of warnings

Reply:
because it starts with `/**`

Reply:
Thanks for the explanation.